todo_app/app.py

```diff
@@ -1,4 +1,9 @@
+import os
 from flask import Flask, render_template, request, redirect, url_for
+from flask_login import LoginManager, UserMixin, login_required, login_user
+import flask_login
```

Comment:
Minor, but I think this import is unnecessary as you use the one the line above

Reply:
Thanks Jack!